src/model.jl

```diff
@@ -97,12 +100,38 @@ function fit_cnmf(data; L=10, K=5, alg=:mult,
         # Record time and loss
         push!(time_hist, time_hist[end] + dur)
         push!(loss_hist, loss)
+
+        # Check convergence
+        if check_convergence && converged(loss_hist, patience, tol)
```

Comment:
Any reason you chose to have this as a `break` statement instead of making it a condition for the `while` loop?

Reply:
No good reason -- just seemed more awkward to have it in the while loop.  The logic of the while loop would have gotten a bit bulky (because `check_convergence` is optional) so this seemed more readable.